### PR TITLE
Guide overview copyedits

### PR DESCRIPTION
**Pull request by chalin**

### CODE REVIEW

File: content/docs/guides/_index.md [hunk 1]
```diff
@@ -73,36 +82,37 @@ message HelloReply {
...
-[protocol buffers GitHub repo](https://github.com/google/protobuf/releases),
-as well as a Go language generator from the [golang/protobuf GitHub
-repo](https://github.com/golang/protobuf), with more languages
-in development. You can find out more in the [proto3 language
-guide](https://developers.google.com/protocol-buffers/docs/proto3) and the
-[reference documentation](https://developers.google.com/protocol-buffers/docs/reference/overview)
-available for each language. The reference documentation also includes a
-[formal specification](https://developers.google.com/protocol-buffers/docs/reference/proto3-spec)
-for the `.proto` file format.
+[protocol buffers GitHub repo][], as well as a Go language generator from the
```
Comment (lucperkins) [inline content/docs/guides/_index.md]: FYI you can omit the `[]` at the end of these links. In this case, `[protocol buffers GitHub repo]` would suffice.

File: content/docs/guides/_index.md [hunk 2]
```diff
@@ -14,14 +14,23 @@ see our [Quick Starts](../quickstart).
 <div id="toc" class="toc mobile-toc"></div>
 
 ### Overview
-In gRPC a client application can directly call methods on a server application on a different machine as if it was a local object, making it easier for you to create distributed applications and services. As in many RPC systems, gRPC is based around the idea of defining a service, specifying the methods that can be called remotely with their parameters and return types. On the server side, the server implements this interface and runs a gRPC server to handle client calls. On the client side, the client has a stub (referred to as just a client in some languages) that provides the same methods as the server.
+
+In gRPC, a client application can directly call a method on a server application
+on a different machine as if it was a local object, making it easier for you to
```
Comment (lucperkins) [inline content/docs/guides/_index.md]: s/was/were

File: content/docs/guides/_index.md [hunk 3]
```diff
@@ -14,14 +14,23 @@ see our [Quick Starts](../quickstart).
...
+languages) that provides the same methods as the server.
 
 ![Concept Diagram](../../img/landing-2.svg)
 
 gRPC clients and servers can run and talk to each other in a variety of environments - from servers inside Google to your own desktop - and can be written in any of gRPC's supported languages. So, for example, you can easily create a gRPC server in Java with clients in Go, Python, or Ruby. In addition, the latest Google APIs will have gRPC versions of their interfaces, letting you easily build Google functionality into your applications.
 
 ### Working with Protocol Buffers
-By default gRPC uses [protocol buffers](https://developers.google.com/protocol-buffers/docs/overview), Google’s
+
+By default gRPC uses [Protocol Buffers][PB docs], Google’s
```
Comment (lucperkins) [inline content/docs/guides/_index.md]: Comma after default

File: content/docs/guides/_index.md (hunk 1 above)
Comment (chalin) [reply content/docs/guides/_index.md]: By suffixing with `[]`, some IDEs (like VS Code) offer edit-time validation of links, which is quite useful! :)

Comment (chalin) [reply content/docs/guides/_index.md]: In that case, do you mind keeping the `[]` suffix?

Comment (lucperkins) [reply content/docs/guides/_index.md]: Good to know! In that case, feel free to disregard. Just a pointer, not a blocker 👍

File: content/docs/guides/_index.md (hunk 2 above)
Comment (chalin) [reply content/docs/guides/_index.md]: Done

File: content/docs/guides/_index.md (hunk 3 above)
Comment (chalin) [reply content/docs/guides/_index.md]: Done

**Pull request merged**